File: page-objects/java-fluentlenium/complete/src/main/java/pages/ShoppingCartPage.java
package pages;

import org.fluentlenium.core.FluentPage;
import org.fluentlenium.core.domain.FluentList;
import org.fluentlenium.core.domain.FluentWebElement;
import org.openqa.selenium.By;

import static org.assertj.core.api.Assertions.assertThat;

public class ShoppingCartPage extends FluentPage {

    private final String PRODUCT_INFO = "z-coast-fjord_link";

    public String getProductBrand() {
        return find(By.className(PRODUCT_INFO)).get(1).text();
    }

    public String getProductName() {
        return find(By.className(PRODUCT_INFO)).last().text();
    }

}

File: page-objects/js-protractor/complete/pages/shopping-cart-page.js

ShoppingCartPage = function ShoppingCartPage() {
    this.productInfo = element.all(by.className('z-coast-fjord_link'));
    this.productBrand = this.productInfo.get(1);
    this.productName = this.productInfo.last();
};

ShoppingCartPage.prototype.getProductName = function() {
    return this.productName.getText();
};

ShoppingCartPage.prototype.getProductBrand = function() {
    return this.productBrand.getText();
};


module.exports = ShoppingCartPage;
